jwst/coron/align_refs_step.py

```diff
@@ -1,6 +1,8 @@
+""" Smooth and align psf image with target image."""
```

Comment:
I would reword this to "Replace bad pixels and align psf ..."

Reply:
Done